pkg/acquisition/modules/loki/loki.go

```diff
@@ -282,6 +282,10 @@ func (l *LokiSource) StreamingAcquisition(out chan types.Event, t *tomb.Tomb) er
 		for {
 			select {
 			case resp := <-respChan:
+				if resp == nil {
```

Comment:
when the channel is closed it will loop indefinitely, one way to check for it is to use the optional boolean returned when reading from a channel:
```go
// ok will be false when the channel is closed
case resp, ok := <-respChan:
    if !ok {
        ll.Warnf("loki channel closed")
        return
    }
```